examples/MagicWeather/ios/MagicWeatherReactNative/AppDelegate.swift

```diff
@@ -4,18 +4,36 @@ import React_RCTAppDelegate
 import ReactAppDependencyProvider
 
 @main
-class AppDelegate: RCTAppDelegate {
-  override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
-    self.moduleName = "MagicWeatherReactNative"
-    self.dependencyProvider = RCTAppDependencyProvider()
+class AppDelegate: UIResponder, UIApplicationDelegate {
+  var window: UIWindow?
 
-    // You can add your custom initial props in the dictionary below.
-    // They will be passed down to the ViewController used by React Native.
-    self.initialProps = [:]
+  var reactNativeDelegate: ReactNativeDelegate?
+  var reactNativeFactory: RCTReactNativeFactory?
```

Comment:
I guess these are unused?

Reply:
good question – it comes directly from the boilerplate RN code so I don't dare touch them (also, not touching the boilerplate code will make it easier to upgrade in the future)